### PR TITLE
Fixed shellcheck to be used from Gobin.

### PR DESCRIPTION
**Pull request by bwplotka**
Somehow for make lint was erroring with:

```
>> linting all of the shell script files
make: /home/bwplotka/Repos/thanos/tmp/bin/shellcheck: Command not found
make: *** [Makefile:303: shell-lint] Error 127
```

Signed-off-by: Bartlomiej Plotka <bwplotka@gmail.com>

### CODE REVIEW

**Comment by bwplotka** [comment]
just guessing now really, ;p

File: Makefile [hunk 1]
```diff
@@ -169,7 +169,7 @@ check-docs: $(EMBEDMD) $(LICHE) build
 .PHONY:shell-format
 shell-format: $(SHFMT)
 	@echo ">> formatting shell scripts"
-	@$(SHFMT) -i 2 -ci -w -s $(shell find . -type f -name "*.sh" -not -path "*vendor*" -not -path "tmp/*")
+	@$(SHFMT)shell -i 2 -ci -w -s $( find . -type f -name "*.sh" -not -path "*vendor*" -not -path "tmp/*")
```
Comment (kakkoyun) [inline Makefile:172]: This doesn't seem right.
`make: /tmp/.bin/shfmt-v3.1.2shell: Command not found`

File: Makefile [hunk 2]
```diff
@@ -231,6 +231,7 @@ install-deps: $(ALERTMANAGER) $(MINIO) $(PROMETHEUS_ARRAY)
 	@echo ">>GOBIN=$(GOBIN)"
 
 .PHONY: docker-ci
+docker-ci: export BIN_DIR= $(shell pwd)/tmp/bin
```
Comment (kakkoyun) [inline Makefile:234]: Why do we need this if you are replacing `BIN_DIR` with `GOBIN`?

File: Makefile (hunk 1 above)
Comment (bwplotka) [reply Makefile:172]: yea this is totally wrong, just typo

File: Makefile (hunk 2 above)
Comment (bwplotka) [reply Makefile:234]: No idea, red herring - although relying on yet another bin dir seems wrong

**Pull request closed without merging**